.changelog/11335.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:enhancement
```

Comment:
How do we handle changelogs for manual backports?

Reply:
I believe the convention is that changelog entries always point to the main PR that closed the issue, not necessarily the PR to fix a failed backport.